docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+# Autopilot Nginx demo
+#
+# For most use cases, it's better to build a custom image using this Nginx base image
+# see https://github.com/autopilotpattern/wordpress for an example
+
+nginx:
+    image: autopilotpattern/nginx
+    restart: always
+    mem_limit: 512m
+    env_file: _env
+    ports:
+        - 80
+    labels:
+        - triton.cns.services=nginx
+
+app:
+    image: autopilotpattern/containerbuddy-demo-app
```

Comment:
Oh, right... it used to part of the ContainerPilot repo but we don't have examples there. Well, one more reason why we should be packaging an example here rather than in some other repo so that you forget about them!

The original app was removed from ContainerPilot in [this commit](https://github.com/joyent/containerpilot/commit/42de688ea535fdb5b070449d22ec1a3ef3ba1843) but the equivalent app lives [in the integration tests](https://github.com/joyent/containerpilot/tree/master/integration_tests/fixtures/app) now.